src/databaseAdapters/mongoose/season.ts

```diff
@@ -0,0 +1,52 @@
+import Season from '../../models/season';
+import { ISeason } from '../../models/types/season';
+import ISeasonDbAdapter from '../databaseInterfaces/season';
+import { RatingBracket } from '../../gameplay/elo/types';
+
+export class MongoSeasonAdapter implements ISeasonDbAdapter {
+  async getCurrentSeason(): Promise<ISeason | null> {
+    const currentSeason: ISeason | null = await Season.findOne({
+      startDate: { $lte: new Date() },
+      endDate: { $gte: new Date() },
```

Comment:
Can you pull `new Date()` into one var above so we can findOne here based on the same date?